src/mats_l1_processing/photometer.py

```diff
@@ -0,0 +1,93 @@
+#%%
```

Comment:
What is this? (same at end of file)

Reply:
Oh I can remove them, they are just for VS code. They make it so that you can run each block as a ipython/jupyter block. Quite usefull.

Reply:
Up to you. I was just curious.